Sources/Mustache/Sequence.swift

```diff
@@ -22,7 +37,7 @@ protocol MustacheSequence {
 
 extension Sequence {
```

Comment:
I'm wondering if this is going to be public if it should be 
```swift
extension MustacheSequence where Self: Sequence {
```
instead of extending `Sequence`

Or alternatively require `MustacheSequence` to conform to `Sequence`.

Reply:
I went with the second suggestion.
A fake conformance to `Sequence` isn't hard anyway, so it's fine even if someone's type isn't actually a `Sequence` but they want to conform it to `MustacheSequence` for some reason.